src/components/input/input.ts

```diff
@@ -108,6 +123,101 @@ export class TextInput extends InputBase {
   /**
    * @private
    */
+  @Input() autocomplete: string;
+
+  /**
+   * @private
+   */
+  @Input() autocorrect: string;
+
+  /**
+   * @private
+   */
+  @Input() autocapitalize: string;
+
+  /**
+   * @private
+   */
+  get autofocus(): boolean {
```

Comment:
I meant this: https://github.com/driftyco/ionic/blob/master/src/components/menu/menu.ts#L238-L249